lib/parallel-api.js

```diff
@@ -8,12 +9,31 @@ var debugGenerator = require('heimdalljs-logger');
 
 var jobs = Number(process.env.JOBS) || require('os').cpus().length;
 
-// create a worker pool using an external worker script
-var pool = workerpool.pool(path.join(__dirname, 'worker.js'), { maxWorkers: jobs });
-
 var loggerName = 'broccoli-persistent-filter:ParallelApi';
 var _logger = debugGenerator(loggerName);
 
+var babelCoreVersion = getBabelVersion();
+
+// return the version of Babel that will be used by this plugin
+function getBabelVersion() {
+  var babelCoreRequirePath = require.resolve('babel-core');
```

Comment:
I believe `require.require('babel-core/package.json').version;` is the correct enchantation to consistently gain access to the version.


`require.resolve('babel-core')` will resolve the packages `main` file, which may reside more nested

Reply:
that worked :)